src/hardware/serialport/softmodem.cpp

```diff
@@ -1008,8 +1009,26 @@ void CSerialModem::Timer2()
 		}
 	}
 
+	// Discard any incoming traffic while in command mode
+	if (commandmode && clientsocket) {
+		size_t usesize = 80;
+		if (!clientsocket->ReceiveArray(tmpbuf, usesize)) {
+			SendRes(ResNOCARRIER);
+			LOG_INFO("SERIAL: No carrier on receive");
+			EnterIdleState();
+		}
+	}
+
+	// Discard any incoming traffic from the waiting client
+	if (waitingclientsocket) {
+		size_t usesize = 80;
```

Comment:
Could this `80` constant be extracted into the outer scope then instead of copy/pasting it? I assume 80 is some important magic number here, and the code would work differently with receiving less or more than 80 bytes max? (I'm really unsure, so I'm asking 😄 )